client/src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from 'react';
+const NavBar = (props) => {
+
+  const [state, setState] = useState('')
```

Comment:
why not null/undefined as initial state?

Reply:
This is used for css property `is-active`. If the user clicks on the nav menu state becomes `is-active`.
If we keep it undefined, we have to check the condition and then make it `''`